src/contour/TerminalSession.cpp

```diff
@@ -616,6 +616,13 @@ void TerminalSession::onHighlightUpdate()
 {
     terminal_.resetHighlight();
 }
+
+void TerminalSession::playSound(terminal::Sequence::Parameters const params_)
+{
+    auto range = params_.range();
+    emit audio.play(params_.at(0), params_.at(1), { range.begin() + 2, range.end() });
```

Comment:
You are constructing a new `std::vector<int>` for the notes here. Is it possible to pass a `span<int>` instead?
If ownership is a problem, we could probably use some reusable pre-allocated vector?
I am just thinking out loud here.

Reply:
`span<int>` has a problem that `params_` gets deallocated before slot gets called, so slot gets garbled values.
A preallocated vector is a nice idea I'll change it to use that.